reporting_app/static/datatable.js

```diff
@@ -17,6 +17,15 @@ var dt_merge_multi_sources_keep_first = function(dt_config){
     );
 }
 
+var dt_merge_lims_container_and_qc_data = function(dt_config){
```

Comment:
`merge_lims_container_and_qc_data` is only ever used via this wrapper function - is it worth having this, or would it be better to just have `merge_lims_container_and_qc_data` take a dt_config object and call it directly in `reporting_app/__init__`?

Reply:
I agree that it adds more code for not very much but since I moved it to the utils I prefer if the function is not made specifically for datatables.
It might be used elsewhere at a latter point.